app/views/galleries/index.atom.builder

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
If you include the `GalleryDisplayingView` concern here with:
```ruby
self.class.send(:include, GalleryDisplayingView)
```

... then you could reuse its `#presenter_for_gallery_image` and `#document_for_gallery_image`.